client/components/domains/contact-details-form-fields/index.jsx

```diff
@@ -537,10 +536,12 @@ export class ContactDetailsFormFields extends Component {
 					? this.renderGAppsFieldset()
 					: this.renderContactDetailsFields() }
 
-				<div className="contact-details-form-fields__extra-fields">{ this.props.children }</div>
+				{ ! React.Children.count( this.props.children ) && (
```

Comment:
I think this reads "if there aren't children, render the children". You can probably just use [the same check as below](https://github.com/Automattic/wp-calypso/pull/46731/files#diff-6f67a370a8ba13172ab358f664f54797ade22756a2253bb974ef06996cdddb12R386).

Reply:
Yeah, that didn't seem to work before but it working now. Thanks!